src/xunit.performance.nuspec

```diff
@@ -54,9 +54,7 @@
         <dependency id="System.Text.Encoding" version="4.0.10" />
         <dependency id="System.Threading" version="4.0.10" />
         <dependency id="System.Threading.Tasks" version="4.0.10" />
-        <dependency id="xunit.abstractions" version="2.0.0" />
-        <dependency id="xunit.extensibility.core" version="2.1.0" />
-        <dependency id="xunit.extensibility.execution" version="2.1.0" />
+        <dependency id="Microsoft.DotNet.BuildTools.TestSuite" version="1.0.0-prerelease-508-01" />
```

Comment:
Sure. Although again this shouldn't affect any other runs as it is adding an additional tfm to search for in the libs. From the cli project.json (https://github.com/dotnet/cli/blob/rel/1.0.0/test/Performance/project.json#L26 ), looks like it will only bring in the dotnet tfm dependencies.